src/pages/index.js

```diff
@@ -1,26 +1,34 @@
 import Head from 'next/head';
 import Image from 'next/image';
 import Link from 'next/link';
+import styled from 'styled-components';
 
-import StyledLink from '../components/Link/StyledLink';
+import Main from '../components/Main';
+import Searchbar from '../components/Searchbar/Searchbar';
 
 const logo = '/logo.png';
 
+const A = styled.a`
```

Comment:
For the "A" you could assign a clearer name

Reply:
For sure. I would keep it simple for the anchor.